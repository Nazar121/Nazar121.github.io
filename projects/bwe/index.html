<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>BWE</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" href="src/css/reset.css">
    <link rel="stylesheet" href="src/css/bootstrap.css">
    <link rel="stylesheet" href="src/css/font-awesome.min.css">
    <link rel="stylesheet" href="src/css/style.css">
    <link rel="stylesheet" href="src/css/slider.css">
    <link rel="stylesheet" href="src/css/slider2.css">
    <link rel="shortcut icon" href="header__logo.png" type="images/x-ico">
</head>
<body>
    
    <div class="wrapper" id="wrapper">
        
        <div class="menu__sm">
            <div class="sm__top">
                <a href="index.html" class="top__logotype"></a>
                <div class="top__openMenu"></div>
                <div class="top__closeMenu"></div>
            </div>
            <nav class="sm__toggleMenu">
                <li>
                    <a href="#wrapper">OUR PROJECTS</a>
                </li>
                <li>
                    <a href="#services">SERVICES</a>
                </li>
                <li>
                    <a href="#about">ABOUT US</a>
                </li>
                <li>
                    <a href="#reviewes">CUSTOMERS</a>
                </li>
                <li>
                    <a href="#contact">CONTACT US</a>
                </li>
            </nav>
        </div>
        
        
        <div class="bgHeader">
            
            <header class="header">
                <div class="container">
                    <div class="row">
                        <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
                            <div class="header__flex">
                                <a href="index.html" class="header__logo"></a>
                                <div class="header__title">
                                    <span class="title__brainMd">brian</span> <span class="title__eardXs">ward</span> electrical
                                </div>
                                <div class="header__subtitle">
                                    <span class="subtitle__electMd">Domestic and commercial electrical</span> services across Scotland
                                </div>
                                <div class="header__button">
                                    Our Projects
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
                
                <div class="header__menuPosition">
                    <!--<div class="container-fluid">
                    <div class="row">
                        <div class="col-lg-12">-->
                            <div class="header__flexNav">
                                <a href="index.html" class="header__logoCopy">
                                    <img src="./src/img/menu__logo.png" width="172" alt="">
                                </a>
                                <nav class="header__menu">
                                    <li>
                                        <a href="#wrapper">OUR PROJECTS</a>
                                    </li>
                                    <li>
                                        <a href="#services">SERVICES</a>
                                    </li>
                                    <li>
                                        <a href="#about">ABOUT US</a>
                                    </li>
                                    <li>
                                        <a href="#reviewes">CUSTOMERS</a>
                                    </li>
                                    <li>
                                        <a href="#contact">CONTACT US</a>
                                    </li>
                                </nav>
                                <nav class="menu__icon">
                                    <li>
                                        <a href="https://www.facebook.com/" target="_blank">
                                            <i class="fa fa-facebook" aria-hidden="true"></i>
                                        </a>
                                    </li>
                                    <li>
                                        <a href="https://twitter.com" target="_blank">
                                            <i class="fa fa-twitter" aria-hidden="true"></i>
                                        </a>
                                    </li>
                                    <li>
                                        <a href="https://linkedin.com" target="_blank">
                                            <i class="fa fa-linkedin" aria-hidden="true"></i>
                                        </a>
                                    </li>
                                </nav>
                            </div>
                        <!--</div>
                    </div>
                </div>-->
                </div>
            </header>
            
        </div>
        
        <section class="services" id="services">
            <div class="container">
                <div class="row">
                    <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
                        <div class="services__flex">
                            <div class="services__border"></div>
                            <div class="services__title">Our Services</div>
                            <div class="services__desc">We undertake a broad range of work in all sectors. From the list below you will see we cover most areas of electrical work in the domestic, commercial and industrial sectors</div>
                        </div>
                    </div>
                </div>
                <div class="row">
                    <div class="col-xs-12 col-sm-12 col-md-4 col-lg-4 servicesBorder">
                        <div class="services__blockCom">
                            <div class="block__pos1-com"></div>
                            <div class="block__pos2-com">
                                <div class="block__subtitle">COMMERCIAL</div>
                                <nav class="block__list">
                                    <li>All types of lighting</li>
                                    <li>DALI lighting</li>
                                    <li>Lutron lighting</li>
                                    <li>Scene setting systems</li>
                                    <li>All types of power systems</li>
                                    <li>Emergency lighting</li>
                                    <li>Fire and smoke alarm systems</li>
                                </nav>
                            </div>
                        </div>
                    </div>
                    <div class="col-xs-12 col-sm-12 col-md-4 col-lg-4 servicesBorder">
                        <div class="services__blockDom">
                            <div class="block__pos1-dom"></div>
                            <div class="block__pos2-dom">
                                <div class="block__subtitle">DOMESTIC</div>
                                <nav class="block__list">
                                    <li>New installations</li>
                                    <li>Complete or part rewiring</li>
                                    <li>Replacement fuse distribution boards</li>
                                    <li>Additional lighting or socket points</li>
                                    <li>All electrical work design and advice</li>
                                    <li>Telephone systems and additional points</li>
                                    <li>Television systems and additional points</li>
                                    <li>Surround sound system installations</li>
                                    <li>Fire and smoke alarm systems</li>
                                    <li>Electric heating</li>
                                    <li>All types of fault finding</li>
                                </nav>
                            </div>
                        </div>
                    </div>
                    <div class="col-xs-12 col-sm-12 col-md-4 col-lg-4 servicesBorder">
                        <div class="services__blockInd">
                            <div class="block__pos1-ind"></div>
                            <div class="block__pos2-ind">
                                <div class="block__subtitle">INDUSTRIAL</div>
                                <nav class="block__list">
                                    <li>All 3 phase installations
</li>
                                    <li>3 phase distribution boards</li>
                                    <li>Industrial lighting</li>
                                    <li>Power designs</li>
                                </nav>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </section>
        
        <div class="bgAbout">
            <section class="abot" id="about">
                <div class="container">
                    <div class="row">
                        <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
                            <div class="about__flex">
                                <div class="about__border"></div>
                                <div class="about__title">ABOUT US</div>
                            </div>
                        </div>
                        <div class="col-xs-12 col-sm-offset-2 col-md-offset-0 col-sm-8 col-md-6 col-lg-offset-1 col-lg-5">
                            <div class="content__slider">
                           <div class="nav-box"> <span class="nav nav-prev"></span> <span class="nav nav-next"></span> </div>
                           <div class="timeline">
                               <div class="timeline-panel"><div class="timeline-item"> <img class="img"> </div></div><div class="timeline-panel"><div class="timeline-item"> <img class="img"> </div></div><div class="timeline-panel"><div class="timeline-item"> <img class="img"> </div></div><div class="timeline-panel"><div class="timeline-item"> <img class="img"> </div></div><div class="timeline-panel"><div class="timeline-item"> <img class="img"> </div></div><div class="timeline-panel"><div class="timeline-item"> <img class="img"> </div></div>
                           </div>
                           <div class="tab-box"> </div>
                       </div>
                        <div class="about__tabs">
                            <div class="tabs__clock"></div>
                            <div class="tabs__text">We offer competitive prices for all works to be carried out</div>
                            <div class="tabs__box"></div>
                        </div>
                        </div>
                        <div class="col-xs-12 col-sm-offset-2 col-md-offset-0 col-sm-8 col-md-6 col-lg-5">
                            <p class="about__desc">B Ward Electrical Ltd is a family based company that has had electricians in its family for over 60 years. It started out as Walker and Ward, with Danny Ward and George Walker, it has always had the same philosophy, <br>
“To do a job correctly as if it were your own”, no exceptions.</p>
                       <p class="about__desc">Mr Ward and Mr Walker have now retired and now past the torch onto myself. I intend to carry on the training and experience that I was luckily enough to receive from them both. B Ward Electrical Ltd has got experience in every circle of life, from installing an outside light to rewiring a Cathedral. In my opinion, all work is good work, and we take the same amount of pride and perfection in every job we do.</p>
                       <div>
                           <p class="about__subtitle">Who we are</p>
                           <p class="about__desc">We are a growing company that offers a fast friendly service, we strive to advise and implement what you, the client wants. We are based in Edinburgh, but have travelled all over Scotland, and the UK. </p>
                       </div>
                       <div>
                           <p class="about__subtitle">What we do</p>
                           <p class="about__desc">We offer a full friendly service from, an initial meeting regarding an estimate for any type of works that is required; to a walk round regarding once the job is complete to make sure you are 100 % satisfied.</p>
                       </div>
                        </div>
                    </div>
                </div>
            </section>
        </div>
        
        <div class="bgQuote">
            <section class="quote" idqoute>
                <div class="container">
                    <div class="row">
                        <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
                            <div class="quote__flex">
                               <div class="quote__info">
                                    <p class="info__text">Commercial &amp; Domestic electrical</p>
                                    <p class="info__text">ContractorS Specialising in LED lighting</p>
                               </div>
                            </div>
                        </div>
                    </div>
                </div>
            </section>
        </div>
        
        <section class="reviewes" id="reviewes">
            <div class="container">
                <div class="row">
                    <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
                        <div class="reviewes__flex">
                            <div class="reviewes__border"></div>
                            <div class="reviewes__title">What Our Customers Say</div>
                            <div class="reviewes__desc">Lorem ipsum pri minimum perfecto legendos ex, tollit dolore nominati pro at, voluptua delectus reprimique ius in. Ad putant latine conceptam mei, euismod omnesque officiis in nam, no nihil aliquid utroque duo.</div>
                        </div>
                    </div>
                    <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
                        <div class="wrap2">
                            <div class="timeline2">
                            <div class="timeline-panel">
                                <div class="timeline-item">
                                    <div class="item__left">
                                        <div class="left__icon">
                                            <div class="icon__img1"></div>
                                        </div>
                                        <h1 class="left__title">You have just invested </h1>
                                        <p class="left__desc">The one of your dreams, and you <br class="desc__md-br1"> take it <br class="desc__lg-br1"> on a trip across Canada.<br class="desc__md-br2"> You only put <br class="desc__lg-br2"> gas in it <br class="desc__sm-br1"> and drive <br class="desc__md-br3"> sightseeing and <br class="desc__lg-br3"> following your <br class="desc__md-br4"> nose. Filters and oil <br class="desc__lg-br4"> changes are <br class="desc__md-br5"> ignored.</p>
                                    </div>
                                    <div class="item__center">
                                        <div class="center__icon">
                                            <div class="icon__img2"></div>
                                        </div>
                                        <h1 class="center__title">You have just invested </h1>
                                        <p class="left__desc">The one of your dreams, and you <br class="desc__md-br1"> take it <br class="desc__lg-br1"> on a trip across Canada.<br class="desc__md-br2"> You only put <br class="desc__lg-br2"> gas in it <br class="desc__sm-br1"> and drive <br class="desc__md-br3"> sightseeing and <br class="desc__lg-br3"> following your <br class="desc__md-br4"> nose. Filters and oil <br class="desc__lg-br4"> changes are <br class="desc__md-br5"> ignored.</p>
                                    </div>
                                    <div class="item__right">
                                        <div class="right__icon">
                                            <div class="icon__img3"></div>
                                        </div>
                                        <h1 class="right__title">You have just invested </h1>
                                        <p class="left__desc">The one of your dreams, and you <br class="desc__md-br1"> take it <br class="desc__lg-br1"> on a trip across Canada.<br class="desc__md-br2"> You only put <br class="desc__lg-br2"> gas in it <br class="desc__sm-br1"> and drive <br class="desc__md-br3"> sightseeing and <br class="desc__lg-br3"> following your <br class="desc__md-br4"> nose. Filters and oil <br class="desc__lg-br4"> changes are <br class="desc__md-br5"> ignored.</p>
                                    </div>
                                </div>
                            </div>
                            <div class="timeline-panel">
                                <div class="timeline-item">
                                    <div class="item__left">
                                        <div class="left__icon">
                                            <div class="icon__img1"></div>
                                        </div>
                                        <h1 class="left__title">You have just invested </h1>
                                        <p class="left__desc">The one of your dreams, and you <br class="desc__md-br1"> take it <br class="desc__lg-br1"> on a trip across Canada.<br class="desc__md-br2"> You only put <br class="desc__lg-br2"> gas in it <br class="desc__sm-br1"> and drive <br class="desc__md-br3"> sightseeing and <br class="desc__lg-br3"> following your <br class="desc__md-br4"> nose. Filters and oil <br class="desc__lg-br4"> changes are <br class="desc__md-br5"> ignored.</p>
                                    </div>
                                    <div class="item__center">
                                        <div class="center__icon">
                                            <div class="icon__img2"></div>
                                        </div>
                                        <h1 class="center__title">You have just invested </h1>
                                        <p class="left__desc">The one of your dreams, and you <br class="desc__md-br1"> take it <br class="desc__lg-br1"> on a trip across Canada.<br class="desc__md-br2"> You only put <br class="desc__lg-br2"> gas in it <br class="desc__sm-br1"> and drive <br class="desc__md-br3"> sightseeing and <br class="desc__lg-br3"> following your <br class="desc__md-br4"> nose. Filters and oil <br class="desc__lg-br4"> changes are <br class="desc__md-br5"> ignored.</p>
                                    </div>
                                    <div class="item__right">
                                        <div class="right__icon">
                                            <div class="icon__img3"></div>
                                        </div>
                                        <h1 class="right__title">You have just invested </h1>
                                        <p class="left__desc">The one of your dreams, and you <br class="desc__md-br1"> take it <br class="desc__lg-br1"> on a trip across Canada.<br class="desc__md-br2"> You only put <br class="desc__lg-br2"> gas in it <br class="desc__sm-br1"> and drive <br class="desc__md-br3"> sightseeing and <br class="desc__lg-br3"> following your <br class="desc__md-br4"> nose. Filters and oil <br class="desc__lg-br4"> changes are <br class="desc__md-br5"> ignored.</p>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div class="tabBox">
                        </div>
                        </div> 
                    </div>
                </div>
            </div>
        </section>
        
        <div class="bgContact">
            <section class="contact" id="contact">
                <div class="container">
                    <div class="row">
                    <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
                        <div class="contact__flex">
                            <div class="contact__border"></div>
                            <div class="contact__title">CONTACT US</div>
                            <div class="contact__desc">Please use the contact form below to get in touch with us. It will be great to hear from you whether we have previously done work for you or this is the first time you have contacted us.</div>
                            <div class="contact__leave">Leave us a message</div>
                        </div>
                    </div>
                    <div class="col-xs-12 col-sm-12 col-md-offset-1 col-md-5 col-lg-offset-2 col-lg-4">
                       <div class="contact__posValid">
                            <input type="text" class="contact__inputName" placeholder="First Name&#042;">
                            <i class="fa fa-check contact__good" aria-hidden="true"></i>
                            <i class="fa fa-times contact__err" aria-hidden="true"></i>
                        </div>
                    </div>
                    <div class="col-xs-12 col-sm-12 col-md-5 col-lg-4">
                        <div class="contact__posValid">
                            <input type="text" class="contact__inputEmail" placeholder="Email&#042;">
                            <i class="fa fa-check contact__good2" aria-hidden="true"></i>
                            <i class="fa fa-times contact__err2" aria-hidden="true"></i>
                        </div>
                    </div>
                    <div class="col-xs-12 col-sm-12 col-md-offset-1 col-md-10 col-lg-offset-2 col-lg-8">
                        <textarea class="contact__inputMessage" placeholder="Message"></textarea>
                    </div>
                    <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
                        <input type="button" value="SEND MESSAGE" class="contact__sendMes">
                    </div>
                </div>
                </div>
            </section>
        </div>
        
        <div class="bgFooter">
            <footer class="footer">
                <div class="footer__copyright">
                    &copy; 2016, brian ward electrical 
                </div>
                <div class="footer__company">
                    <!--<span class="company__designXs">Design by <span>OTAKOYI</span></span> <img src="./src/img/footer__company.png" alt="">-->
                </div>
            </footer>
        </div>
        
        <a href="#wrapper" class="toUp"></a>
        
    </div>
    
    
    <script src="src/js/jquery-3.1.0.min.js"></script>
    <script src="src/js/bootstrap.js"></script>
    <script src="src/js/slider.js"></script>
    <script src="src/js/slider2.js"></script>
    <script src="src/js/main.js"></script>
</body>
</html>