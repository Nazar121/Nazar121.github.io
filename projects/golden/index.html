<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <title>Golden</title>
    
    <link rel="stylesheet" href="dist/libs/bootstrap/bootstrap.min.css">
    <link rel="stylesheet" href="dist/libs/font-awesome/css/font-awesome.min.css">
    <link rel="stylesheet" href="dist/css/reset.min.css">
    <link rel="stylesheet" href="dist/css/animations.min.css">
    <link rel="stylesheet" href="dist/css/main.min.css">
    
    <link rel="shortcut icon" href="ico.jpg" type="images/x-ico">
    
</head>
<body>
    
    <!-- Header -->
    <div class="animatedParent">
        <header class="header" id="home">
            <div class="fixedMenu">
                <div class="container">
                    <div class="row justify-content-between align-items-center">
                        <div class="col-auto">
                            <a href="index.html" class="logo"></a>
                        </div>
                        <div class="col-auto">
                            <div class="d-block d-md-none">
                                <div class="toggleSmallMenu">
                                    <i class="fa fa-bars" aria-hidden="true"></i>
                                    <i class="fa fa-times" aria-hidden="true"></i>
                                </div>
                            </div>
                            <nav class="navScroll menu d-none d-md-flex">
                                <a href="#home">HOME</a>
                                <a href="#services">SERVICES</a>
                                <a href="#portfolio">PORTFOLIO</a>
                                <a href="#aboutUs">ABOUT</a>
                                <a href="#contactUs">CONTACT</a>
                            </nav>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-12">
                            <nav class="navScroll smallMenu">
                                <a href="#home">HOME</a>
                                <a href="#services">SERVICES</a>
                                <a href="#portfolio">PORTFOLIO</a>
                                <a href="#aboutUs">ABOUT</a>
                                <a href="#contactUs">CONTACT</a>
                            </nav>
                        </div>
                    </div>
                </div>
            </div>
            <div class="container header__content animated bounceIn">
                <div class="row">
                    <div class="col">
                        <div class="content__welcome">Welcome To Our Studio!</div>
                        <div class="content__title">IT'S NICE TO MEET YOU</div>
                        <a href="#aboutUs" class="content__more">TELL ME MORE</a>
                    </div>
                </div>
            </div>
        </header>
    </div>
    
    <!-- Services -->
    <section class="services" id="services">
        <div class="container">
            <div class="row animatedParent">
                <div class="col animated growIn">
                    <div class="title">SERVICES</div>
                    <div class="titleDesc">Proin iaculis purus consequal sem cure.</div>
                </div>
            </div>
        </div>
        <div class="container">
            <div class="row animatedParent animateOnce" data-sequence='500'>
                <div class="col-md-4 services__item animated growIn" data-id='1'>
                    <img src="dist/img/services_img1.png" class="item__img" alt="services_img1">
                    <div class="item__title">E-Commerce</div>
                    <div class="item__desc">
                        Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.
                    </div>
                </div>
                <div class="col-md-4 services__item animated growIn" data-id='2'>
                    <img src="dist/img/services_img2.png" class="item__img" alt="services_img2">
                    <div class="item__title">Responsive Web</div>
                    <div class="item__desc">
                        Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.
                    </div>
                </div>
                <div class="col-md-4 services__item animated growIn" data-id='3'>
                    <img src="dist/img/services_img3.png" class="item__img" alt="services_img3">
                    <div class="item__title">Web Security</div>
                    <div class="item__desc">
                        Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.
                    </div>
                </div>
            </div>
        </div>
    </section>
    
    <!-- Portfolio -->
    <section class="portfolio" id="portfolio">
        <div class="container">
            <div class="row animatedParent">
                <div class="col animated growIn">
                    <div class="title">OUR PORTFOLIO</div>
                    <div class="titleDesc">Proin iaculis purus consequal sem cure.</div>
                </div>
            </div>
            <div class="row animatedParent animateOnce" data-sequence='500'>
                <div class="col-sm-6 col-md-4 animated growIn" data-id='1'>
                    <a href="404.html" target="_blank" class="work">
                        <div class="work__img"><img src="dist/img/portfolio_img1.jpg" alt="portfolio_img1"></div>
                        <figcaption>
                            <p class="work__desc">Ebony &amp; Lvory</p>
                            <p class="work__type">Branding</p>
                        </figcaption>
                    </a>
                </div>
                <div class="col-sm-6 col-md-4 animated growIn" data-id='2'>
                    <a href="404.html" target="_blank" class="work">
                        <div class="work__img"><img src="dist/img/portfolio_img2.jpg" alt="portfolio_img2"></div>
                        <figcaption>
                            <p class="work__desc">Smart Stationary</p>
                            <p class="work__type">Print Design</p>
                        </figcaption>
                    </a>
                </div>
                <div class="col-sm-6 col-md-4 animated growIn" data-id='3'>
                    <a href="404.html" target="_blank" class="work">
                        <div class="work__img"><img src="dist/img/portfolio_img3.jpg" alt="portfolio_img3"></div>
                        <figcaption>
                            <p class="work__desc">Clever Poster</p>
                            <p class="work__type">Print Design</p>
                        </figcaption>
                    </a>
                </div>
                <div class="col-sm-6 col-md-4 animated growIn" data-id='4'>
                    <a href="404.html" target="_blank" class="work">
                        <div class="work__img"><img src="dist/img/portfolio_img4.jpg" alt="portfolio_img4"></div>
                        <figcaption>
                            <p class="work__desc">Vinyl Record</p>
                            <p class="work__type">Product Mock-Up</p>
                        </figcaption>
                    </a>
                </div>
                <div class="col-sm-6 col-md-4 animated growIn" data-id='5'>
                    <a href="404.html" target="_blank" class="work">
                        <div class="work__img"><img src="dist/img/portfolio_img5.jpg" alt="portfolio_img5"></div>
                        <figcaption>
                            <p class="work__desc">Treehouse Template</p>
                            <p class="work__type">Web Design</p>
                        </figcaption>
                    </a>
                </div>
                <div class="col-sm-6 col-md-4 animated growIn" data-id='6'>
                    <a href="404.html" target="_blank" class="work">
                        <div class="work__img"><img src="dist/img/portfolio_img6.jpg" alt="portfolio_img6"></div>
                        <figcaption>
                            <p class="work__desc">Burned Logo</p>
                            <p class="work__type">Branding</p>
                        </figcaption>
                    </a>
                </div>
            </div>
        </div>
    </section>
    
    <!-- About Us -->
    <section class="aboutUs" id="aboutUs">
        <div class="container">
            <div class="row animatedParent">
                <div class="col animated growIn">
                    <div class="title">ABOUT US</div>
                    <div class="titleDesc">Proin iaculis purus consequal sem cure.</div>
                </div>
            </div>
            <div class="animatedParent animateOnce" data-sequence='500'>
                <div class="row history animated fadeInLeft" data-id='1'>
                   <div class="col-12 history__img--smallDevices">
                        <img src="dist/img/aboutUs_img1.png" class="history__img" alt="">
                    </div>
                    <div class="col-12 col-lg-4 history__right">
                        <h2 class="history__title">July 2010</h2>
                        <h2 class="history__title">Our Humble Beginnings</h2>
                        <p class="history__text">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.</p>
                    </div>
                    <div class="col-12 col-lg-4 history__img--bigDevices">
                        <img src="dist/img/aboutUs_img1.png" class="history__img" alt="">
                    </div>
                </div>
                <div class="row history justify-content-end animated fadeInRight" data-id='2'>
                    <div class="col-lg-4">
                        <img src="dist/img/aboutUs_img2.png" class="history__img" alt="">
                    </div>
                    <div class="col-lg-4 history__left">
                        <h2 class="history__title">January 2011</h2>
                        <h2 class="history__title">Facing Startup Battles</h2>
                        <p class="history__text">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
                    </div>
                </div>
                <div class="row history animated fadeInLeft" data-id='3'>
                   <div class="col-12 history__img--smallDevices">
                        <img src="dist/img/aboutUs_img3.png" class="history__img" alt="">
                    </div>
                    <div class="col-lg-4 history__right">
                        <h2 class="history__title">December 2012</h2>
                        <h2 class="history__title">Enter The Dark Days</h2>
                        <p class="history__text">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni.</p>
                    </div>
                    <div class="col-12 col-lg-4 history__img--bigDevices">
                        <img src="dist/img/aboutUs_img3.png" class="history__img" alt="">
                    </div>
                </div>
                <!-- More history -->
                <div class="history--more">
                    <div class="row history justify-content-end animated fadeInRight" data-id='4'>
                    <div class="col-12 col-lg-4">
                        <img src="dist/img/aboutUs_img4.png" class="history__img" alt="">
                    </div>
                    <div class="col-12 col-lg-4 history__left">
                        <h2 class="history__title">February 2014</h2>
                        <h2 class="history__title">Our Triumph</h2>
                        <p class="history__text">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean.</p>
                    </div>
                </div>
                </div>
                <div class="row history justify-content-center animated fadeInUp" data-id='5'>
                    <div class="col-md-4">
                        <img src="dist/img/aboutUs_img5.png" class="history__img history__img--more" alt="">
                    </div>
                </div>
            </div>
        </div>
    </section>
    
    <!-- Team -->
    <section class="team">
        <div class="container">
            <div class="row animatedParent">
                <div class="col animated growIn">
                    <div class="title">UOR AMAZING TEAM</div>
                    <div class="titleDesc">Proin iaculis purus consequal sem cure.</div>
                </div>
            </div>
            <div class="animatedParent animateOnce" data-sequence='500'>
                <div class="row justify-content-center">
                    <div class="col-sm-6 col-md-4 animated growIn" data-id='1'>
                        <figure class="friend">
                            <div class="friend__img"><img src="dist/img/team_img_1.jpg" alt=""></div>
                            <figcaption>
                                <p class="friend__name">Name Surname</p>
                                <p class="friend__profession">Marketer</p>
                            </figcaption>
                            <div class="friend__socials">
                                <a href="https://twitter.com/" target="_blank" class="social"><i class="fa fa-twitter" aria-hidden="true"></i></a>
                                <a href="https://www.facebook.com/" target="_blank" class="social"><i class="fa fa-facebook" aria-hidden="true"></i></a>
                                <a href="https://www.pinterest.com" target="_blank" class="social"><i class="fa fa-pinterest-p" aria-hidden="true"></i></a>
                            </div>
                        </figure>
                    </div>
                    <div class="col-sm-6 col-md-4 animated growIn" data-id='2'>
                        <figure class="friend">
                            <div class="friend__img"><img src="dist/img/team_img_2.jpg" alt=""></div>
                            <figcaption>
                                <p class="friend__name">Name Surname</p>
                                <p class="friend__profession">Coder</p>
                            </figcaption>
                            <div class="friend__socials">
                                <a href="https://twitter.com/" target="_blank" class="social"><i class="fa fa-twitter" aria-hidden="true"></i></a>
                                <a href="https://www.facebook.com/" target="_blank" class="social"><i class="fa fa-facebook" aria-hidden="true"></i></a>
                                <a href="https://www.pinterest.com" target="_blank" class="social"><i class="fa fa-pinterest-p" aria-hidden="true"></i></a>
                            </div>
                        </figure>
                    </div>
                    <div class="col-sm-6 col-md-4 animated growIn" data-id='3'>
                        <figure class="friend">
                            <div class="friend__img"><img src="dist/img/team_img_3.jpg" alt=""></div>
                            <figcaption>
                                <p class="friend__name">Name Surname</p>
                                <p class="friend__profession">Graphic Designer</p>
                            </figcaption>
                            <div class="friend__socials">
                                <a href="https://twitter.com/" target="_blank" class="social"><i class="fa fa-twitter" aria-hidden="true"></i></a>
                                <a href="https://www.facebook.com/" target="_blank" class="social"><i class="fa fa-facebook" aria-hidden="true"></i></a>
                                <a href="https://www.pinterest.com" target="_blank" class="social"><i class="fa fa-pinterest-p" aria-hidden="true"></i></a>
                            </div>
                        </figure>
                    </div>
                </div>
                <div class="row justify-content-center animated growIn" data-id='4'>
                    <div class="col-12 col-lg-8">
                        <div class="team__desc">
                            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos aliquid deleniti, nihil explicabo. Voluptate unde, tempora, veniam mollitia debitis cupiditate tenetur illum incidunt est officiis adipisci rerum deserunt fugiat. Laboriosam!
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>
    
    <!-- Logos -->
    <section class="logos">
        <div class="container">
            <div class="row justify-content-center">
                <div class="col-auto">
                    <img src="dist/img/logos_img1.jpg" class="logo" alt="logos_img1">
                </div>
                <div class="col-auto">
                    <img src="dist/img/logos_img2.jpg" class="logo" alt="logos_img2">
                </div>
                <div class="col-auto">
                    <img src="dist/img/logos_img3.jpg" class="logo" alt="logos_img3">
                </div>
                <div class="col-auto">
                    <img src="dist/img/logos_img4.jpg" class="logo" alt="logos_img4">
                </div>
            </div>
        </div>
    </section>
    
    <!-- Contact Us -->
    <section class="contactUs" id="contactUs">
        <div class="container">
            <div class="row animatedParent">
                <div class="col animated growIn">
                    <div class="title title--white">CONTACT US</div>
                    <div class="titleDesc">Proin iaculis purus consequal sem cure.</div>
                </div>
            </div>
        </div>
        <form class="form">
            <div class="container">
                <div class="row">
                    <div class="col-md-6">
                         <div class="form-group">
                            <input type="text" name="name" class="form-control" placeholder="YOUR NAME *">
                          </div>
                          <div class="form-group">
                            <input type="text" name="email" class="form-control" placeholder="YOUR E-MAIL *">
                          </div>
                          <div class="form-group">
                            <input type="text" name="subject" class="form-control" placeholder="SUBJECT">
                          </div>
                    </div>
                    <div class="col-md-6">
                        <textarea name="message" class="form-control" placeholder="MESSAGE"></textarea>
                    </div>
                </div>
                <div class="row">
                    <div class="col">
                        <button type="button" class="btn btn-warning">SEND MESSAGE</button>
                    </div>
                </div>
            </div>
        </form>
    </section>
    
    <!-- Footer -->
    <footer class="footer">
        <div class="container">
            <div class="row align-items-center">
                <div class="col-md-6">
                    <p>Copyright 2017 FreebiesXpress.com</p>
                </div>
                <div class="col-md-6">
                    <div class="sociales">
                        <div class="social"><i class="fa fa-twitter" aria-hidden="true"></i></div>
                        <div class="social"><i class="fa fa-facebook" aria-hidden="true"></i></div>
                        <div class="social"><i class="fa fa-pinterest-p" aria-hidden="true"></i></div>
                        <div class="social"><i class="fa fa-glide-g" aria-hidden="true"></i></div>
                    </div>
                </div>
            </div>
        </div>
    </footer>
    
    <!-- To Up -->
    <nav class="toUp">
        <a href="#home">
            <i class="fa fa-chevron-up" aria-hidden="true"></i>
        </a>
    </nav>
    
    <script src="dist/libs/jquery/jquery.min.js"></script>
    <script src="dist/libs/bootstrap/bootstrap.min.js"></script>
    <script src="dist/js/css3-animate-it.min.js"></script>
    <script src="dist/js/main.min.js"></script>
</body>
</html>